### PR TITLE
Updated for FRITZ-Dect 500 devices

### PR DESCRIPTION
**Pull request by SmokingDevices**
While playing around with this lib i noticed that some of the newer devices are not supported.

### CODE REVIEW

File: src/main/java/com/github/kaklakariada/fritzbox/mapping/Deserializer.java [hunk 1]
```diff
@@ -39,6 +39,7 @@ public Deserializer() {
     }
 
     public <T> T parse(String data, Class<T> resultType) {
+        //LOG.info("parsing data:"+data);
```
Comment (kaklakariada) [inline src/main/java/com/github/kaklakariada/fritzbox/mapping/Deserializer.java]: Please don't add commented code
```suggestion
```

File: src/test/java/com/github/kaklakariada/fritzbox/mapping/DeserializerTest.java [hunk 2]
```diff
@@ -30,8 +30,29 @@
 public class DeserializerTest {
 
     @Test
-    public void parseDeviceList() throws IOException {
-        final String fileContent = Files.readAllLines(Paths.get("src/test/resources/deviceListPayload.xml")).stream()
+    public void parseDeviceListFritzDect200() throws IOException {
+        final String fileContent = Files.readAllLines(Paths.get("src/test/resources/deviceListConnectedFritzDect200Payload.xml")).stream()
+                .collect(joining("\n"));
+        new Deserializer().parse(fileContent, DeviceList.class);
```
Comment (kaklakariada) [inline src/test/java/com/github/kaklakariada/fritzbox/mapping/DeserializerTest.java:36]: Thanks for adding tests 👍

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/kaklakariada/fritzbox/mapping/Deserializer.java (hunk 1 above)
Comment (SmokingDevices) [reply src/main/java/com/github/kaklakariada/fritzbox/mapping/Deserializer.java]: has been removed, sorry for that.

**Review by kaklakariada — APPROVED** [review]

**Comment by kaklakariada** [comment]
I will build a new release soon, maybe on the weekend.

**Comment by kaklakariada** [comment]
Published 1.2.1 to jcenter: https://jcenter.bintray.com/com/github/kaklakariada/fritzbox-java-api/1.2.1/

**Pull request merged**